dd-sdk-android-core/src/test/kotlin/com/datadog/android/core/internal/logger/SdkInternalLoggerTest.kt

```diff
@@ -477,6 +482,171 @@ internal class SdkInternalLoggerTest {
         }
     }
 
+    @ParameterizedTest
+    @ValueSource(floats = [0.0f, 0.3f])
```

Comment:
you can use `forge` to generate random float

Reply:
The goal here is to check null and non null parameters. (I pass null in case if creationSampleRate == 0.0) and do not copy-paste whole test for specific value, not sure `forge` is applicable here